packages/cx-editors/src/components/RichText/tokens/cxRichText.ts

```diff
@@ -109,26 +111,43 @@ const Default = asCxTokenSpec()({
   },
 });
 
-const Copyright = asCxTokenSpec()({
+const Basic = asCxTokenSpec()({
   ...Default,
+  Core: pick(Default.Core, 'paragraph', 'Bold', 'Underline', 'Link', 'SuperScript'),
+  Components: pick(Default.Components, 'Bold', 'Underline', 'SuperScript'),
+});
+
+const Copyright = asCxTokenSpec()({
+  ...Basic,
   Spacing: {
     paragraph: 'mx-9 py-9 md:mx-0 md:mb-4 md:p-0 lg:mt-2 lg:mb-0 lg:py-0',
```

Comment:
The spacing should go on the copyright slot in footer not here.

Reply:
Spacing changed to the copyright slot